### ISSUE TITLE
Binary cross entropy

### ISSUE THREAD

**Issue by monosakkarid** [manashmndl/DeadSimpleSpeechRecognizer#8]
Thanks for the create code. I am just wondering why you use binary cross entropy loss function in your github code and categorical cross-entropy in your blog post.  I have more than 2 categories I wish to classify, and it seems strange to me that you are using binary cross entropy for a classification problem with 3 classes. Can you explain to me why you are using this loss function?

**Comment by manashmandal** [comment]
![image](https://user-images.githubusercontent.com/8012876/56093351-83045180-5ee9-11e9-8767-b250ab791f0f.png)
I used categorical cross entropy in both cases.
